### DISCUSSION TITLE
Does Umami collect any statistics from the self-hosted instances?

### DISCUSSION THREAD

**Discussion by marekbrze** [umami-software/umami#2062] [General]
Hello,

I have a quick questions about self-hosted instances - does Umami store any data of users that visit my website? 

I know that a lot of tools use Sentry or other software to catch bugs and they collect ip addresses etc. I need to review this info and add it to the privacy policy if needed. 

Also is there option to opt-out from this kind of analytics?

**Comment by mikecao** [comment]
Umami collects very limited telemetry data about your system, but we do not collect any data about your users. To opt-out of telemetry, you can set the environment variable, `DISABLE_TELEMETRY`. See https://umami.is/docs/environment-variables
